authorization_services/topics/enforcer-keycloak-enforcement-filter.adoc

```diff
@@ -88,12 +88,27 @@ Requests are allowed even when there is no policy associated with a given resour
 +
 *** *DISABLED*
 +
-Completely disables the evaluation of policies and allows access to any resource.
+Completely disables the evaluation of policies and allows access to any resource. When `enforcement-mode` is `DISABLED`
+applications are still able to obtain all permissions granted by {project_name} through the <<_enforcer_authorization_context, Authorization Context>>
 +
 ** *on-deny-redirect-to*
 +
 Defines a URL where a client request is redirected when an "access denied" message is obtained from the server. By default, the adapter responds with a 403 HTTP status code.
 +
+** *path-cache*
++
+Defines how the policy enforcer should track associations between paths in your application and resources defined in {project_name}. The cache
+is responsible to avoid unnecessary requests to a {project_name} server by caching associations between paths and protected resources.
++
```

Comment:
Do we mean the cache "is a responsible way to avoid unnecessary requests" or that the cache "is needed to avoid..."?

Reply:
Thanks. I suggest changing the sentence to "The cache is needed to avoid..." and we are good.